eth/backend.go

```diff
@@ -217,6 +217,9 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	if config.OverrideVerkle != nil {
 		overrides.OverrideVerkle = config.OverrideVerkle
 	}
+	if config.OverrideStoryNostoi != nil {
+		overrides.OverrideStoryNostoi = config.OverrideStoryNostoi
```

Comment:
Need to set balance of precompile contracts.

```suggestion
		overrides.OverrideStoryNostoi = config.OverrideStoryNostoi
		evm.StateDB.AddBalance(common.HexToAddress("0x1B"), uint256.NewInt(1), tracing.BalanceChangeTouchAccount)
```